### ISSUE TITLE
Don't have add-in in Apps for Office/Shared Folder after Sideload Office Add-ins

### ISSUE THREAD

**Issue by salingun** [OfficeDev/office-js-docs-pr#421]
I finish all step of this guide. Create folder Manifest, set  Catalog Url:  \\\PCNAME\Manifest and restart application but nothing show up in  Apps for Office&gt;Shared Folder. Does anybody know this issue?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0ddce0bd-5c66-c2cb-0640-ee4d5bd401c5
* Version Independent ID: b5d4bc26-a43e-4a85-e7d5-fe4fc4ec3d0d
* Content: [Sideload Office Add-ins for testing](https://docs.microsoft.com/en-us/office/dev/add-ins/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins#)
* Content Source: [docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md)
* Product: **office 365**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by Reezaali** [comment]
Hi @salingun. What's the version & build number of your version of Office? Have you added the `<Requirements> `element to your manifest file? See [here](https://docs.microsoft.com/en-us/office/dev/add-ins/develop/specify-office-hosts-and-api-requirements#set-the-requirements-element-in-the-manifest) for more information.

**Comment by salingun** [comment]
Thanks your reply. I go to Setting/Account/About Project and see version is 15.0.4420.1017, I try to add <Requiments> to manifest file but issue still there, nothing show up in shared folder
![image](https://user-images.githubusercontent.com/21010188/46165567-275ed780-c2bb-11e8-86c5-c291fdb58022.png)
I I just created a project in VS 2015 and when start it, it catch error message: _**"None of the apps in your project can be activated in the target. The manifest file of one or more apps contain API sets or Office applications that are not supported by the target Office client. To debug those apps, update manifest files to exclude any unsupported API sets or Office applications, and then start the project again. Alternatively, you can debug your apps by using Office 365 as a target."**_

**Comment by salingun** [comment]
I'm trying to install the service pack as this guide https://docs.microsoft.com/vi-vn/office/dev/add-ins/develop/update-your-javascript-api-for-office-and-manifest-schema-version. I'm not sure it can be fixed, but I think I need to do something with it

**Comment by Reezaali** [comment]
Thanks for replying @salingun. Can you try removing the `<Requirements>` element from your manifest file? After doing that, close and re-open Project. Thanks!

**Comment by salingun** [comment]
I tried removing, close and re-open Project many time but shared folder is still the same. The office version 15.0.4420.1017 is fine to run and test add-in? I think the problem is related to the error that appears when running App for office project on VS **_"None of the apps in your project can be activated in the target. The manifest file of one or more apps contain API sets or Office applications that are not supported by the target Office client. To debug those apps, update manifest files to exclude any unsupported API sets or Office applications, and then start the project again. Alternatively, you can debug your apps by using Office 365 as a target."_**

**Comment by Reezaali** [comment]
Hi @salingun, is it possible to share the content of your manifest file with us? You can anonymize urls or other private information as needed. Thanks!

**Comment by Reezaali** [comment]
Hi @salingun, I've confirmed that the version of Project that you listed above does not support the Web add-in model.

**Comment by Reezaali** [comment]
I've updated the docs with the version support information [here](https://github.com/OfficeDev/office-js-docs-pr/commit/5aa2201f1cc13300fb8c94554392d3116bb3755b#diff-3159520ee2a84d4f634bacc37288d469)

**Comment by salingun** [comment]
Thanks for replying @Reezaali , I will update version of Project to run it again, I will get back to you for result. Thanks!

**Comment by salingun** [comment]
Hi @Reezaali , to day I update ms office & ms project in my pc. It don't have 15.1 version, I found some topic remind update sp1 of office and project, I did it . After that, ms office and project updated to 15.0.4569.1506, and my add-in can work normally. Here I pick the link to download sp1 package if someone need it later. Thanks for your support @Reezaali 
SP1 Office: https://www.microsoft.com/en-US/download/details.aspx?id=42006
SP1 Project: https://www.microsoft.com/en-us/download/details.aspx?id=42020
![image](https://user-images.githubusercontent.com/21010188/46228487-6dd03700-c38c-11e8-8550-cb689311ec25.png)
